java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -766,6 +767,15 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_castTo(JNIEnv *env, job
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_byteListCast(JNIEnv *env, jobject j_object,
+                                                                jlong handle, jboolean endianness_config) {
+  JNI_NULL_CHECK(env, handle, "native handle is null", 0);
+  cudf::column_view *column = reinterpret_cast<cudf::column_view *>(handle);
```

Comment:
We need the `try` catch that we have in the other JNI APIs.  JNI is C and if an exception escapes the entire JVM exits because it does not know what to do with it.

We also need

```
cudf::jni::auto_set_device(env);
```

To guard against java threading issues.